crates/biome_css_parser/src/syntax/mod.rs

```diff
@@ -87,7 +104,28 @@ pub(crate) fn parse_rule_block(p: &mut CssParser) -> ParsedSyntax {
     CssDeclarationList.parse_list(p);
     p.expect(T!['}']);
 
-    Present(m.complete(p, CSS_BLOCK))
+    Present(m.complete(p, CSS_DECLARATION_LIST_BLOCK))
+}
+
+#[inline]
+pub(crate) fn parse_or_recover_rule_list_block(p: &mut CssParser) -> RecoveryResult {
+    parse_rule_list_block(p).or_recover(
+        p,
+        &ParseRecovery::new(CSS_BOGUS_BLOCK, BODY_RECOVERY_SET).enable_recovery_on_line_break(),
+        expected_block,
+    )
+}
+#[inline]
+pub(crate) fn parse_rule_list_block(p: &mut CssParser) -> ParsedSyntax {
```

Comment:
I'd like to introduce a new block kind. I'm not sure what is the best way to implement it, however I think it's better to extend them in future.

Examples:
https://drafts.csswg.org/css-syntax-3/#typedef-block-contents
https://www.w3.org/TR/css-syntax-3/#typedef-declaration-list

It's interesting that `swc` has a generic block which can contains any element.
https://github.com/swc-project/swc/blob/37bb0e3bde97f96bc79771e9d15cc1f130566a50/crates/swc_css_ast/src/base.rs#L158-L232